examples/tfengine/resources_only.hcl

```diff
@@ -106,6 +106,14 @@ template "resources" {
           member = "group:example-data-viewers@example.com"
         }]
       }]
+      topic_message_retention_duration = [{
```

Comment:
This should generate the tf files here https://github.com/GoogleCloudPlatform/solution-acceleration-toolkit/tree/master/examples/tfengine/generated

Reply:
https://github.com/GoogleCloudPlatform/solution-acceleration-toolkit/blob/8ae9b6385f25151d26cf8e64d844d02796e779fe/examples/tfengine/modules/team.hcl#L484

Add the config here.

Reply:
> This should generate the tf files here https://github.com/GoogleCloudPlatform/solution-acceleration-toolkit/tree/master/examples/tfengine/generated

The new diff has the generated main.tf file.